Gemfile

```diff
@@ -17,6 +17,7 @@ gem 'dotenv-rails', '~> 2.7'
 gem 'rails', '~> 5.0.7'
 
 # Needed for XML serialization of ActiveRecord::Base
+gem "activejob-uniqueness", "~> 0.2.0"
```

Comment:
How about?

```suggestion
gem 'activejob-uniqueness', require: 'active_job/uniqueness/sidekiq_patch'
```

Reply:
I thought I would not use that for now. I do not think we need to unlock the jobs via the sidekiq-web and adding too much burden on queue. See https://github.com/veeqo/activejob-uniqueness/issues/16

- ttl is only 1 day, so lock will be released after 1 day. Let see first if we have issues with that
- if there is an issue I prefer we do it with a console command for now